### PR TITLE
Update Jenkinsfile to follow archetypes

### PR DESCRIPTION
**Pull request by debayangg**
### Testing done
Haven't older jdks 11 and 17.

```[tasklist]
### Submitter checklist
- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue
```

### CODE REVIEW

**Comment by jonesbusy** [comment]
It doesn't look like https://github.com/jenkinsci/bitbucket-kubernetes-credentials-plugin/pull/172/files 

Can you use the archetype one ? We can safely remove old java version

**Review by jonesbusy — CHANGES_REQUESTED** [review]
Use archetype file from  https://github.com/jenkinsci/archetypes/blob/master/common-files/Jenkinsfile

File: Jenkinsfile [hunk 1]
```diff
@@ -3,9 +3,11 @@
  https://github.com/jenkins-infra/pipeline-library/
 */
 buildPlugin(
+ forkCount: '1C', // run this number of tests in parallel for faster feedback.  If the number terminates with a 'C', the value will be multiplied by the number of available CPU cores
   useContainerAgent: true, // Set to `false` if you need to use Docker for containerized tests
   configurations: [
     [platform: 'linux', jdk: 11],
     [platform: 'linux', jdk: 17],
     [platform: 'linux', jdk: 21],
+    [platform: 'windows', jdk: 17],
```
Comment (jonesbusy) [inline Jenkinsfile:12]: Windows doesn't make any sense here. It's a plugin that will work only on Kubernetes linux nodes

**Review by jonesbusy — CHANGES_REQUESTED** [review]

**Comment by debayangg** [comment]
have updated to follow archetype and removed the windows jdk

**Review by jonesbusy — APPROVED** [review]

**Pull request merged**